### DISCUSSION TITLE
Typewriter Sounds?

### DISCUSSION THREAD

**Discussion by Valdus92** [KDE/ghostwriter#872] [Ideas]
Call me old and shallow. But when I write the sound of a typewriter is the sound of accomplishment. 

This is one of the few things that would make GW perfect for me (does a perfect writing app even exist)?

How much effort would it take to have the bliss of the ticky-tack?

**Comment by abenson** [comment]
There are plenty of applications that can do this independent of the
editor. For example, I've used `bucklespring`.

**Reply by Valdus92** [reply]
I tried installing bucklespring on my chromebook. It installed but running it is posing a problem.

**Comment by jason-mehmel** [comment]
Focuswriter might be what you need!
